container_pipeline/utils.py

```diff
@@ -211,3 +211,29 @@ def complete(self):
             os.remove(self._path)
             self.logger.info('Removing lock file for {} at {}'.format(
                 self.name, self._path))
+
+
+def form_targetfile_link(git_URL, git_path, git_branch, target_file):
+    """
+    Helper function to generate Dockerfile link.
+    :param git_URL: The url of the git repository.
+    :param git_path: The path, relative to the git repository root,
+                     where the file resides.
+    :param git_branch: The repository branch where the file resides.
+    :param target_file: The name of the target file.
+    :return: The reachable link to the dockerfile.
+    """
+
+    # TODO : Move this into a lib.
```

Comment:
If this is a new function being added and since the `TODO` suggests moving it to a different file, what's blocking us from doing it upfront?

Reply:
Yes, i need to remove that TODO.